test/parallel/test-util-inspect.js

```diff
@@ -668,9 +669,8 @@ if (typeof Symbol !== 'undefined') {
   subject = [1, 2, 3];
   subject[Symbol('symbol')] = 42;
 
-  assert.strictEqual(util.inspect(subject), '[ 1, 2, 3 ]');
-  assert.strictEqual(util.inspect(subject, options),
-                     '[ 1, 2, 3, [length]: 3, [Symbol(symbol)]: 42 ]');
```

Comment:
This assertion is removed because we already have a test above that shows that symbol key properties are visible with `showHidden`.